### ISSUE TITLE
Zombie processes

### ISSUE THREAD

**Issue by PeterNashaat** [openvstorage/framework#753]
Every now and then we get an ovs module warning about one or more zombie processes ,sometimes it goes away and other times we have to kill the parent process to get rid of the zombie processes

**Comment by wimpers** [comment]
Hi Peter,

can you provide some more info? Which processes, can you add relevant logs?

**Comment by PeterNashaat** [comment]
Hi wim
like now in cpu-05 be-scale-3 

```
root@cpu-05:~# ps aux | grep 'Z'
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
ovs      11117  0.0  0.0      0     0 ?        Zs   14:30   0:00 [ssh] <defunct>
ovs      11118  0.0  0.0      0     0 ?        Zs   14:30   0:00 [ssh] <defunct>
```

and for the parent Processe 

```
root@cpu-05:~# ps aux | grep 17655
root     16442  0.0  0.0  11748  2220 pts/9    S+   14:50   0:00 grep --color=auto 17655
ovs      17655  0.0  0.2 760380 71600 ?        S    Jul31   0:42 python2.7 /usr/bin/celery worker -A ovs.celery_run -l info -Q ovs_generic,ovs_vJzYlMYEPAWj0xYN --autoscale=8,2 -Ofair

```

**Comment by wimpers** [comment]
@PeterNashaat do you know which action triggered it  or is able to reproduce this? Is this Fargo or Eugene?

**Comment by PeterNashaat** [comment]
It keeps happening in all environments ,and it's on du-conv-3 right now ,and this is a Eugene env
but we don't know how to reproduce it 

```
root@du-conv-3-01:~# ps aux | grep 'Z'
USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
ovs      16946  0.0  0.0      0     0 ?        Zs   10:30   0:00 [ssh] <defunct>
ovs      16951  0.0  0.0      0     0 ?        Zs   10:30   0:00 [ssh] <defunct>
```

and the Parent 

```
root@du-conv-3-01:~# cat /proc/16951/status | grep -i ppid
PPid:   16555
root@du-conv-3-01:~# ps aux | grep 16555
ovs      16555  0.1  0.0 689136 61352 ?        S    10:29   0:03 python2.7 /usr/bin/celery worker -A ovs.celery_run -l info -Q ovs_generic,ovs_Hp5mmtLSgOpmtHmm,ovs_masters --autoscale=8,2 -Ofair

```

**Comment by wimpers** [comment]
@pploegaert can you check if this also happens on our autotest env? Can we keep a eye on this on the OVH env?

**Comment by khenderick** [comment]
Put to verification since it might have been fixed by #820

**Comment by JeffreyDevloo** [comment]
### Information

Found zombieprocesses on the Fargo environment that I installed for testing purposes right after the install (lingering SSH just as in described in the issue).
Could not find the same zombie processes since #820 had been packaged.

### Steps
- Fresh install of a cluster
- Checked for zombie processes

### Output

Found no zombie processes for the three installs that I did

### Test result

Test passed.
I have based my test result on the Fargo installation that would always leave a ssh zombie process.

### Packages
- openvstorage                       2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage
- openvstorage-backend               1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin
- openvstorage-backend-core          1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps       1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin Web Applications
- openvstorage-cinder-plugin         1.2.2-rev.38.dcc3b76-1                                   amd64        OpenvStorage Cinder plugin for OpenStack
- openvstorage-core                  2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage core
- openvstorage-hc                    1.7.4-rev.766.0d9f30e-1                                  amd64        openvStorage Backend plugin HyperConverged
- openvstorage-sdm                   1.6.4-rev.388.1f4375d-1                                  amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps               2.7.4-rev.4159.0707c77-1                                 amd64        openvStorage Web Applications
